Mail/smtpmx.php

```diff
@@ -125,24 +125,31 @@ class Mail_smtpmx extends Mail {
     /**
      * Switch to test mode - don't send emails for real
      *
-     * @var boolean $debug
+     * @var boolean $test
      */
     var $test = false;
 
     /**
      * Turn on Net_SMTP debugging?
      *
-     * @var boolean $peardebug
+     * @var boolean $debug
      */
     var $debug = false;
 
+    /**
+     * Set debug_handler on Net_SMTP
+     *
+     * @var string $debug_handler
```

Comment:
Wasn't sure if this is a string or a callable. Didn't look into it though. But otherwise lgtm! 👍

Reply:
@till Thanks. Yes, it is a callable. I changed that.